src/platform/api.py

```diff
@@ -35,13 +46,25 @@ def _make_get_request(endpoint: str) -> dict:
 
     url = urljoin(_get_domain_name(), f"api/{endpoint}")
 
-    response = requests.get(
-        url,
-        headers=headers,
-    )
+    if verb == "GET":
+        response = requests.get(
+            url,
+            headers=headers,
+        )
+    elif verb == "POST":
+        response = requests.post(
+            url,
+            headers=headers,
+        )
```

Comment:
Perhaps nicer if you use [`requests.request`](https://requests.readthedocs.io/en/latest/api/#requests.request)?

Reply:
lol much nicer :)

Reply:
fixed